acceptance/config/nodes/hosts.yaml

```
@@ -12,10 +12,10 @@ HOSTS:
       - classifier
       - default
     docker_cmd: '["/sbin/init"]'
-  centos7:
-    platform: el-7-x86_64
+  debian12:
+    platform: debian-12-x86_64
     hypervisor: docker
-    image: centos:7
+    image: debian:12
     roles:
       - agent
     docker_cmd: '/usr/sbin/sshd -D -E /var/log/sshd.log'
```

I wonder why this is sshd and for el8/9 it's `/sbin/init`. In `beaker-hostgenerator` we always use `/sbin/init` so I'd guess this should be
```suggestion
    docker_cmd: '["/sbin/init"]'
```

And then also drop `use_image_entrypoint`?

---

thanks, done